app/models/agents/pushover_agent.rb

```diff
@@ -108,6 +108,7 @@ def working?
     def send_notification(post_params)
       response = HTTParty.post(API_URL, query: post_params)
       puts response
+      log "Sent notification with message \"#{post_params['message']}\" to \"#{post_params['user']}\""
```

Comment:
Good idea, maybe we could log all params but the token? `post_params.except('token').inspect` should be human readable enough, I think.